### PR TITLE
Nojira | @ericbakenhus | Fix deploy

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Actually fix deploy issues
    - Revert bad options change
    - Move a filter that was preventing expired stories from being removed from Algolia

# Review By (Date)
- Soon/Retro

# Criticality
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at code?

### CODE REVIEW

File: netlify/functions/handle-deploy.mts [hunk 1]
```diff
@@ -70,9 +70,7 @@ export default async (req: Request) => {
       oauthToken: process.env.STORYBLOK_MANAGEMENT_OAUTH_TOKEN,
     });
 
-    const storyRes = await storyblokManagement.get(`/spaces/${data.space_id}/stories/${data.story_id}`, {
-      version: data.action === 'published' ? 'published' : 'draft',
-    });
+    const storyRes = await storyblokManagement.get(`/spaces/${data.space_id}/stories/${data.story_id}`);
```
Comment (ericbakenhus) [inline netlify/functions/handle-deploy.mts:73]: Forgot the management API doesn't support those options and just returns everything.

File: netlify/functions/handle-deploy.mts [hunk 2]
```diff
@@ -111,18 +109,18 @@ export default async (req: Request) => {
...
+
       console.log('Event folder (un)publish event detected. Deploying...');
       if (run) {
         await fetch(deployUrl, { method: 'POST' });
       }
       console.log('Deploy triggered');
     }
 
-    storiesToProcess = storiesToProcess.filter((story) => story.full_slug.startsWith('events/sync/'));
-
```
Comment (ericbakenhus) [inline netlify/functions/handle-deploy.mts:125]: Moving this inside the event folder conditional block. Only in that specific case do we want to be upserting/deleting stories specifically in the event/sync folder.

**Comment by ericbakenhus** [comment]
Need to figure out better ways to test these functions...

**Comment by sherakama** [comment]
> Need to figure out better ways to test these functions...

We can chat more about this in our Monday developer chat

**Pull request merged**